_data/package_index.yml

```
@@ -1375,3 +1375,17 @@
   description: Object-oriented Fortran falling block game using Curses console graphics
   categories: examples
   tags: game
+
+- name: gFTL
+  github: Goddard-Fortran-Ecosystem/gFTL
+  description: Templates for  STL-like Fortran implementations of containers for arbitrary types (Vector, Set, Map, Deque, Stack, Queue, ...)
+  categories: containers 
+  tags: container template
+  license: Apache-2.0 
+  
+  - name: gFTL
```

The `name` field seems incorrect. Perhaps a bad copy pasta? :spaghetti:

---

```suggestion
  - name: yaFyaml
```
Yes - sorry about that.   Could have sworn I had updated that field.   Probably should not have done this while listening to the talk.  :-)